src/wallet/rpcwallet.cpp

```diff
@@ -3224,7 +3229,10 @@ static UniValue fundrawtransaction(const JSONRPCRequest& request)
 
     CAmount fee;
     int change_position;
-    FundTransaction(pwallet, tx, fee, change_position, request.params[1]);
+    CCoinControl coin_control;
+    // Automatically select (additional) coins. Can be overriden by options.add_inputs.
+    coin_control.m_add_inputs = true;
```

Comment:
This is already the default value in `coin_control` so maybe remove this?

Reply:
I think it adds clarity that this (default) value matters for this caller.

Reply:
e5327f947c310849e1ddbb24321e4c9f85564549

nit
```cpp
// By default new inputs are added automatically. Can be overriden by options.add_inputs.
CHECK_NONFATAL(coin_control.m_add_inputs);
```